Cargo.toml

```diff
@@ -44,6 +44,8 @@ members = [
 	"ecosystem-modules/starport",
 	"ecosystem-modules/compound-cash",
 	"ecosystem-modules/stable-asset/lib/stable-asset",
+
+	"gamedao/sense",
```

Comment:
actually there needs to be a gamedao/sense, as it would interface with zero/sense on the zero chain via ocw